ethpm/dependencies.py

```diff
@@ -0,0 +1,41 @@
+from typing import Dict, List
+
+
+class Dependencies:
+    """
+    Access build dependencies belonging to a package.
+    """
+
+    def __init__(self, build_dependencies: Dict[str, "Package"]) -> None:
+        self.build_dependencies = build_dependencies
+
+    def __getitem__(self, key: str) -> "Package":
+        return self.get(key)
+
+    def __contains__(self, key: str) -> bool:
+        return key in self.build_dependencies
+
+    def _validate_name(self, name: str) -> None:
+        if name not in self.build_dependencies:
+            raise KeyError(
+                "Package name: {0} not found in build dependencies.".format(name)
+            )
+
+    def get(self, key: str) -> "Package":
```

Comment:
The object you've created here closely maps to a dictionary, but the behavior of this method would be surprising as it raises a `KeyError` if the dependency is not present.  Normal dictionaries just return `None` on a call to `get` with a missing key.

I think the most appropriate thing to do would be dropping the `.get` method since `get_dependency_package` holds more semantic meaning and performs the same task.